src/pages/roadmap/Section.tsx

```diff
@@ -0,0 +1,40 @@
+import clsx from 'clsx'
+import { StyledLink } from 'src/components/StyledLink'
+import styles from './Section.module.scss'
+
+type SectionProps = {
+    title: React.ReactElement,
+    icon: string,
+    iconPosition: 'left' | 'right',
+    subtitle: string,
+    descriptionText: string,
+    descriptionLink: string,
+    projects: string[],
+    className?: string,
+}
+
+export const Section: React.FC<SectionProps> = ({
+    title,
+    icon,
+    iconPosition,
+    subtitle,
+    descriptionText,
+    descriptionLink,
+    projects,
+    className
+}) => {
+    return <div className={clsx(styles.container, iconPosition==='right' && styles.reverse, className)}>
+        <div className={styles.iconWrapper}>
+          <div className={styles.icon} >
+            <img src={icon} alt=''/>
+          </div>
+        </div>
+        <div className={styles.detailWrapper}>
+            <div>{title}</div>
+            <div className={styles.divider}/> 
+            <div className={styles.subtitle}>{subtitle}</div>
+            <div className={styles.descriptionText}>{descriptionText}{' '}<StyledLink className={styles.link} href={''} colored >{descriptionLink}</StyledLink></div>
```

Comment:
> href={''}

Is this expected?

Reply:
yes, there are some plaveholders on this page, some link hrefs are not decided yet.

Reply:
Maybe we should add a TODO

Reply:
added by https://github.com/Magickbase/nervos-official-website/pull/268/commits/bf60af25437b415a1459a0fceee9fee8ff6423ae